coalib/collecting/Collectors.py

```diff
@@ -177,6 +177,23 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer=None,
         bears_found[index].append(bear)
         bear_globs_with_bears.add(glob)
 
+    unused_globs = set(bear_globs) - set(bear_globs_with_bears)
+    suffix_globs = {}
+
+    for glob in unused_globs:
+        if glob is not '**' and glob is not '*':
+            if glob.endswith('bear'):
```

Comment:
I think you will need to use `#pragma nt: no cover` here

I assume this is because of filename casing on windows, but it might specifically be caused by other coala code adding case-insensitivity to work around windows.

In any case, this is worth a pragma no cover.